roles/adaravis/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Tasks for adaravis role
+
+- name: Ensure aravis rpm package is installed
+  ansible.builtin.dnf:
+    name: aravis
+    state: present
+
+- name: Find arv-tool executable
+  ansible.builtin.find:
+    paths: /opt/aravis/bin/
+    patterns: 'arv-tool-*'
+    file_type: file
+  register: arv_tool_search
+
+- name: Set arv-tool path fact
+  ansible.builtin.set_fact:
+    adaravis_arv_tool: >-
+      {{ arv_tool_search.files[0].path if arv_tool_search.matched > 0
+      else '/opt/aravis/bin/arv-tool-0.8' }}
```

Comment:
This can actually be removed, the `aravis-rpm` was updated to install to `/usr/local/bin`, so it is automatically in the path. Also, it now creates a softlink `arv-tool` that points to the binary, so we don't need to discover the version number.

Reply:
Fixed in https://github.com/NSLS2/nsls2.ioc_deploy/pull/37/commits/91c2cff349d960ce7c3895c30c8b9d4354bdbf0d